libtransmission/bandwidth.cc

```diff
@@ -206,210 +172,162 @@ static void phaseOne(tr_ptrArray const* peerArray, tr_direction dir)
          * out in a timely manner. */
         size_t const increment = 3000;
```

Comment:
`increment` could be constexpr now. Probably won't make any difference but :man_shrugging: